grpc-sys/build.rs

```diff
@@ -11,15 +11,111 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#![allow(dead_code, unused_imports)]
```

Comment:
You can use submodule like following to fix the warnings:

#[cfg(feature)]
mod imp {}

#[cfg(not(feature))]
mod imp {}

fn main() {
    imp::build_and_link()
}